src/main/resources/static/ng-bits/current-container.html

```diff
@@ -5,10 +5,10 @@
 </div>
 
 <div class="panel-body row">
-    <div class="col-sm-10 row">
-        <div class="field col-sm-6"><label>Type</label> <span>{{c.type | serverTypeLabel}}</span></div>
-        <div class="field col-sm-6"><label>Version</label> <span>{{c.version}}</span></div>
-        <div class="field col-sm-6" ng-show="hasConnectInfo(c)">
+    <div class="col-xs-10 row">
```

Comment:
Actually `xs` probably shouldn't be used here. That one is special in that it overrides the responsive, collapsing behavior of Bootstrap grids:

http://getbootstrap.com/css/#grid-options

however, I say this without having yet tested these latest changes with an iPhone.